lib/dealer.rb

```diff
@@ -0,0 +1,31 @@
+class Dealer
+
+  attr_reader :attributes
+
+  def initialize(attributes)
+    @attributes = attributes
+  end
+
+  def new_deck
+    cards.shuffle
+  end
+
+  def get_card(id)
+    cards[id]
```

Comment:
since you're storing the `id` on all your Card objects, it might be worth doing a [detect](http://apidock.com/ruby/Enumerable/detect) on the id itself, instead of just cueing off the position of a card in the array. If not, there's no reason to store the "id" on Card.